code/nodes/dataPreprocessing/funcs.q

```diff
@@ -91,3 +91,7 @@ dataPreprocessing.nullEncode:{[feat;func]
    flip 0^(func each flip feat)^flip[feat],names!nullValues
    ]
   }
+
+
+// Temporary infreplace function until toolkit is update
+dataPreprocessing.infreplace:{.ml.infreplace["f"$x]}
```

Comment:
I think this function should use the infinity replace outlined in the current master version of AutoML i.e. at https://github.com/cmccarthy1/automl-1/blob/master/code/updml.q